types/mempool/sender_nonce.go

```diff
@@ -65,6 +89,9 @@ func (snm *senderNonceMempool) setSeed(seed int64) {
 // Insert adds a tx to the mempool. It returns an error if the tx does not have at least one signer.
 // priority is ignored.
 func (snm *senderNonceMempool) Insert(_ sdk.Context, tx sdk.Tx) error {
+	if snm.txCount >= snm.maxTx {
```

Comment:
Unfortunately this change doesn't honor turning off the max with setting it to 0 or less than 0; can we perhaps do
```go
if snm.maxTx >= 0 && snm.txCount >= snm.maxTx {
```

but in the current form still looks good to me since the default value is used hence it'll be bounded by default.

Reply:
@odeke-em, thanks for the review!!!! 
I originally had a boolean for this. However, I like the idea of it being negative means unbounded as well. What do you think, @tac0turtle @kocubinski?

Reply:
makes sense to me